docs/design-reviews/decision-log/examples/memory/Decision-Log.md

```diff
@@ -0,0 +1,10 @@
+# Decision Log
+
+This document is used to track key decisions that are made during the course of the project.
+This can be used at a later stage to understand why decisions were made and by whom.
+
+| **Decision** | **Date** | **Alternatives Considered** | **Reasoning** | **Detailed doc** | **Made By** | **Work Required** |
```

Comment:
I know we had more decisions in our log - It would be great to add more lines here.